src/osp/Active/SysNewton.h

```diff
@@ -236,6 +219,50 @@ class SysNewton : public IDynamicSystem
      */
     static Vector3 get_rigidbody_CoM(ACompNwtBody const& body);
 
+    /**
+     * Compute the volume of a part
+     *
+     * Traverses the immediate children of the specified entity and sums the
+     * volumes of any detected collision volumes. Cannot account for overlapping
+     * collider volumes.
+     * 
+     * @param rScene [in] ActiveScene containing relevant scene data
+     * @param part   [in] The part
+     * 
+     * @return The part's collider volume
+     */
+    static float compute_part_volume(ActiveScene& rScene, ActiveEnt part);
```

Comment:
I'm a bit confused by what "part" refers to. It's clearly not a vehicle part; should this be rigid body or just body?

Reply:
Any child of a rigidbody, which has collider children. The rigidbody function is separate, although it may be possible to combine them